src/metabase/models/collection.clj

```diff
@@ -179,8 +181,9 @@
   'Explode' a `location-path` into a sequence of Collection IDs, and parse them as integers. THIS DOES NOT VALIDATE
   THAT THE PATH OR RESULTS ARE VALID. This unchecked version exists solely to power the other version below."
   [location-path]
-  (for [^String id-str (rest (str/split location-path #"/"))]
-    (Integer/parseInt id-str)))
+  (if (= location-path "/")
+    []
+    (mapv #(Integer/parseInt %) (rest (str/split location-path #"/")))))
```

Comment:
[`Integer/.parseInt`](https://clojure.org/news/2024/09/05/clojure-1-12-0#qualified_methods)? 🤣

Reply:
The new interop is nice, but since there are a few arities I believe it will result in dynamic dispatch.

Reply:
I didn't notice you've bumped to 1.12 already:) Anyway, `parse-long` shall do too.